apis/swagger.yml

```diff
@@ -517,7 +517,7 @@ paths:
         - name: "filters"
           in: "query"
           description: |
-            Filters encoded as JSON string(type map[string][]string in Golang). This API will list containers match all of the filters. For example, `{"status": ["paused"]}` will only return paused containers.
+            Filters encoded as JSON string(type map[string]map[string]bool in Golang). This API will list containers match all of the filters. For example, `{"status": ["paused"]}` will only return paused containers.
```

Comment:
I am wondering if we have supported all these filter options. We should try to make the swagger definition and code logic consistent. cc @Ace-Tang for check.

For the un-implemented filter option or option in plan, we can mark TODO to tell readers.